validator/src/main.rs

```diff
@@ -1621,11 +1621,14 @@ pub fn main() {
         validator_config.accounts_hash_interval_slots,
     ) {
         eprintln!(
-            "Invalid snapshot configuration provided: snapshot intervals are incompatible. \n\t- \
-             full snapshot interval MUST be a multiple of incremental snapshot interval (if \
-             enabled) \n\t- full snapshot interval MUST be larger than incremental snapshot \
-             interval (if enabled) \nSnapshot configuration values: \n\tfull snapshot interval: \
-             {} \n\tincremental snapshot interval: {}",
+            "Invalid snapshot configuration provided: snapshot intervals are incompatible. \
+             \n\t- full snapshot interval MUST be a multiple of incremental snapshot interval (if \
+             enabled)\
+             \n\t- full snapshot interval MUST be larger than incremental snapshot \
+             interval (if enabled)\
+             \nSnapshot configuration values:\
+             \n\tfull snapshot interval: {}\
+             \n\tincremental snapshot interval: {}",
```

Comment:
`fmt` adds a leading space on the lines after the opening quote, which we're losing here. Super nitty, but restore those spaces?

Reply:
I might be misunderstanding which item you're referring to, but I think I retained the spaces you're referring to. Here is a screen cap from my editor; the lines as part of the string literal after the first line have a leading space in comparison to the `if` statement directly below:
<img width="120" alt="image" src="https://github.com/solana-labs/solana/assets/5400107/1c91e51d-8882-4556-b0e0-5c8228697fdc">

Reply:
Hmm, strange; yesterday I swore github was not rendering that additional space. But it does indeed look okay now. Sorry for the 🤪